Common/SimConfig/include/SimConfig/ConfigurableParamHelper.h

```diff
@@ -24,13 +26,49 @@ namespace o2
 namespace conf
 {
 
+// Utility structure for passing around ConfigurableParam data member info
+// (where value is the string representation)
+struct paramDataMember {
+  std::string name;
+  const char* value;
+  std::string provenance;
+
+  std::string toString(bool showProv) const
```

Comment:
if possible put this into a source (cxx) file and avoid including <iostream> in headers. Use <iofwd> for forward declarations.